import "common.proto";

//841 set
message RunSetRequest{
    required int32 hero_no = 1;
    required int32 runt_type = 2;
    required int32 runt_po = 3;
    required int32 runt_id = 4;
}

message RunSetResponse{
    required CommonResponse res = 1;
}

//842
message RuntPickRequest{
    required int32 hero_no = 1;
    required int32 runt_type = 2;
    optional int32 runt_id = 3;
}

message RuntPickResponse{
    required CommonResponse res = 1;
}
